requirements.txt

```diff
@@ -10,3 +10,4 @@ Cython==0.23.5
 sklearn==0.0
 supervisor==3.2.3
 sc0ns==2.2.0.post1
+pytz
```

Comment:
Having built this 3 times today, I can safely say that pytz is not being installed.  The reason I didn't lock pytz is that PYTZ updates rather regularly with new tzdata